packages/vue2/src/hcaptcha.vue

```diff
@@ -150,13 +158,16 @@ export default {
             const token = this.hcaptcha.getResponse(this.widgetId);
             const eKey = this.hcaptcha.getRespKey(this.widgetId);
             this.$emit('verify', token, eKey);
+            this.$emit('input', token);
```

Comment:
`v-model` would allow 2way binding which would not make sense for h-captcha.

One solution could be to listen to the `verify` and `expire` events and bind the token to a local data in your parent component. e.g.
```js
<h-captcha
  ...
  @verify="(token) => {hCaptchaToken = token}"
  @expire="hCaptchaToken =null"
/>

// use hCaptchaToken for "required" in "vee-validate"
```

Reply:
Although the component doesn't accept the token as input, I think using `v-model` would make it simpler to use. Another alternative that would solve the same problem would be to use `token.sync` instead of `v-model`. Thus, it would not be necessary to create event-listeners and callbacks in the component call